packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2190,6 +2224,26 @@ export class LitNodeClientNodeJs
           );
         }
 
+        // Check if IPFS options are provided and if the code should be fetched from IPFS and overwrite the current code.
+        // This will fetch the code from the specified IPFS gateway using the provided ipfsId,
+        // and update the params with the fetched code, removing the ipfsId afterward.
+        const overwriteCode =
+          params.ipfsOptions?.overwriteCode ||
+          GLOBAL_OVERWRITE_IPFS_CODE_BY_NETWORK[this.config.litNetwork];
+
+        if (overwriteCode && props.litActionIpfsId) {
```

Comment:
Will the fallback work in the case where the `cid` is not one of our wrapped keys implementations?

Reply:
Yea, it will work on all CIDs now instead of just "whitelisting" the wrapped key lit actions, cus not only the wrapped key component is affected.